### PR TITLE
[AND-4963] adapter changes to support in-feed videos

### PR DESCRIPTION
**Pull request by HuFangshuai**
pr for ticket: https://vungle.atlassian.net/browse/AND-4963

### CODE REVIEW

File: Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java [hunk 1]
```diff
@@ -403,19 +407,65 @@ private void updateUserPrivacySettings(final MaxAdapterParameters parameters)
...
         }
-        else if ( adFormat == MaxAdFormat.LEADER )
-        {
-            return BannerAdSize.BANNER_LEADERBOARD;
+
+        int deviceWidthPx = getApplicationWindowWidth( context );
+        return AppLovinSdkUtils.pxToDp( context, deviceWidthPx );
+    }
+
+    private VungleAdSize vungleAdSize(
```
Comment (ashinagawa) [inline Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java:406]: Do we support custom size here?

Comment (HuFangshuai) [reply Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java:406]: No, because MAX guide not pass fixed width and height to us.

Comment (ashinagawa) [reply Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java:406]: iOS side, we can pass custom size with code below. Can you do similar for Android side? 

```
// Custom Banner Size request
CGFloat height = 300;
CGFloat width = 400;
self.adView.frame = CGRectMake(0, 0, width, height);
[self.adView setExtraParameterForKey: @"adaptive_banner" value: @"false"];
[self.adView setLocalExtraParameterForKey: @"adaptive_banner_width" value: [[NSNumber numberWithFloat:width] stringValue]];
[self.adView setLocalExtraParameterForKey: @"adaptive_banner_height" value: [[NSNumber numberWithFloat:height] stringValue]];
```

Comment (ashinagawa) [reply Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java:406]: And, this is not on their integration guide tho.

File: Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java [hunk 2]
```diff
@@ -403,19 +403,44 @@ private void updateUserPrivacySettings(final MaxAdapterParameters parameters)
...
-    private static BannerAdSize vungleAdSize(final MaxAdFormat adFormat)
+    private VungleAdSize vungleAdSize(
+        final MaxAdFormat adFormat,
+        final MaxAdapterParameters parameters)
     {
-        if ( adFormat == MaxAdFormat.BANNER )
+        final Map<String, Object> localExtraParameters = parameters.getLocalExtraParameters();
+        Object isAdaptiveObj = localExtraParameters.get( "adaptive_banner" );
+        boolean isAdaptiveBanner = false;
+        if ( isAdaptiveObj instanceof String ) {
```
Comment (ashinagawa) [inline Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java:413]: Do you still support adaptive here?

Comment (HuFangshuai) [reply Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java:413]: No, just to check adaptive_banner should be false for MAX custom size.
I have no idea if we need to check this field or is it ok only check the adaptive_banner_width and adaptive_banner_height? This depend on MAX integration guide doc.

**Review by ManojBudumuru-vungle — APPROVED** [review]

**Pull request merged**